src/App.tsx

```diff
@@ -15,48 +17,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  byAsc,
+  byLength,
+  byDefault,
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortBy: SortType,
+  isReversed: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  preparedGoods.sort((good1, good2) => {
+    switch (sortBy) {
+      case SortType.byAsc:
+        return good1.localeCompare(good2);
+      case SortType.byLength:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  });
```

Comment:
let's add a check if SortType is equal to byDefault to avoid unnecessary sorting